src/components/TestToolsModal.tsx

```diff
@@ -46,6 +53,18 @@ function TestToolsModal({isTestToolsModalOpen = false}: TestToolsModalProps) {
                 </Text>
                 <ProfilingToolMenu />
                 <ClientSideLoggingToolMenu />
+                {!!shouldStoreLogs && (
+                    <TestToolRow title={translate('initialSettingsPage.troubleshoot.debugConsole')}>
+                        <Button
+                            small
+                            text={translate('initialSettingsPage.debugConsole.viewConsole')}
+                            onPress={() => {
+                                toggleTestToolsModal();
+                                Navigation.navigate(ROUTES.SETTINGS_CONSOLE.getRoute(Navigation.getActiveRoute()));
```

Comment:
```suggestion
                                Navigation.navigate(ROUTES.SETTINGS_CONSOLE.getRoute(ROUTES.HOME));
```

Reply:
That is true. I assumed you can't open the debug menu web (can you?), but even if you can't, let's keep `Navigation.getActiveRoute()` since it's more universal. Thanks!